toponymy/topic_naming.py

```diff
@@ -420,10 +419,10 @@ def __init__(
         self.documents = documents
         self.document_vectors = document_vectors
         self.document_map = document_map
-        if (cluster_layers is not None) and (type(cluster_layers).__name__
-                                             != "ClusterLayers"):
+        if (cluster_layers
+                is not None) and not isinstance(cluster_layers, ClusterLayers):
             raise ValueError(
-                f"cluster_layers must be of type ClusterLayers class not {type(cluster_layers).__name__}"
+                f"cluster_layers must be of type ClusterLayers not {type(cluster_layers)}"
```

Comment:
Why remove the `.__name__`? It avoids the awkward `<type BLAH>` string representation of most classes.